plugins/modules/aci_bd.py

```diff
@@ -1,6 +1,8 @@
 #!/usr/bin/python
 # -*- coding: utf-8 -*-
 
+# Copyright: (c) 2017, Jacob McGill (@jmcgill298)
+# Copyright: (c) 2024, Samita Bhattacharjee (@samitab) <samitab@cisco.com>
```

Comment:
Had a quick look at child objects and see that there are also options missing for those. Did you look into those and should we include class like fvRsABDPolMonPol, fvRsBDToFhs, fvRsIgmpsn, fvRsMldsn? Also noticed some non relation classes that could be attached pimBD (with its filter policies), igmpIfP? Did not configure everything so more might be missing. Should we add this to the module?

Reply:
@akinross and @lhercot - Should we create a separate issue for all relevant missing properties of child objects and do it as a separate PR instead? I am only conscious that this PR was done to resolve this particular issue #534 and also because it'll be a significant addition.

Reply:
I personally think that creating issue for each missing option is a bit of overkill, and would add them to this PR.

Reply:
Oh sorry, I meant only one issue for all the other missing options not for each option individually. However, if its more simpler to keep it in the same PR will just do it in this PR. 😄